_episodes/l1-03-tools-II.md

```diff
@@ -65,21 +66,21 @@ options with which to fiddle.
 > > ```python
 > > x = {"a": 37, "b": 42, "c": 927}
 > > y = "hello " + "world"
-> > 
-> > 
+> >
+> >
 > > class foo(object):
 > >     def f(self):
 > >         z = 3
 > >         return y ** 2
-> > 
-> >     def g(self, x: int, y: int = 42) -> int:
```

Comment:
I like type annotations. I think they are part of good coding practices in python, but probably not in an introductory course, specially if we are not using them consistently in the examples. Maybe, again, mention them if just wit a sentence and a link for them to check out on their own?

Reply:
Yeah, I think few users will be familiar with them and believe these examples should be as simple as possible. We do mention mypy in the linters section so I'll add a bit more info about type hints there.